src/box/relay.cc

```diff
@@ -533,6 +517,7 @@ relay_final_join_f(va_list ap)
 	auto guard = make_scoped_guard([=] { relay_exit(relay); });
 
 	relay_cord_init(relay);
+	say_info("final join started");
```

Comment:
Another debug print.

Reply:
This print is used in `replication_luatest/shutdown_test.lua`. But we decided it should be written another way and this version of test is just dropped. So the log is dropped too.